api/v1beta1/verticaautoscaler_types.go

```diff
@@ -70,7 +71,7 @@ type VerticaAutoscalerSpec struct {
 	// here it will be used as a prefix for the new subcluster.  Otherwise, we
 	// use the name of this VerticaAutoscaler object as a prefix for all
 	// subclusters.
-	Template Subcluster `json:"template"`
+	Template v1.Subcluster `json:"template"`
```

Comment:
Let's v1beta1.Subcluster for v1beta1 VerticaAutoscaler.Template, for backward compatibility. Next PI we plan to remove v1beta1 VerticaAutoscaler too anyway.